app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError,
+)
```

Comment:
According to the checklist, you should use `( )` when importing multiple classes/modules. But in Python, it is more common to use `,` without parentheses. The parentheses are used when the line exceeds the maximum length (79 or 120 characters depending on the style guide). So, you can remove the parentheses if the line is not too long.